src/Materials/Textures/texture.ts

```diff
@@ -764,9 +764,9 @@ export class Texture extends BaseTexture {
             if (typeof this._buffer === "string" && (this._buffer as string).substr(0, 5) === "data:") {
                 serializationObject.base64String = this._buffer;
                 serializationObject.name = serializationObject.name.replace("data:", "");
-            } else if (this.url && StartsWith(this.url, "data:") && this._buffer instanceof Uint8Array) {
+            } else if (this.url && StringTools.StartsWith(this.url, "data:") && this._buffer instanceof Uint8Array) {
                 serializationObject.base64String = "data:image/png;base64," + EncodeArrayBufferToBase64(this._buffer);
-            } else if (Texture.ForceSerializeBuffers) {
+            } else if (Texture.ForceSerializeBuffers || (this.url && StartsWith(this.url, "blob:"))) {

```

Comment:
Is it expected that you use `StringTools.StartsWith` two lines above and only `StartsWith` here?

Reply:
nope, this is wrong and should be fixed. Thanks!!
In the future we will use eslint to make sure those legacy imports are not used.